PercolationStats.java

```diff
@@ -1,13 +1,72 @@
+import edu.princeton.cs.algs4.StdOut;
 import edu.princeton.cs.algs4.StdRandom;
 import edu.princeton.cs.algs4.StdStats;
 import edu.princeton.cs.algs4.WeightedQuickUnionUF;
 
 public class PercolationStats {
-   public PercolationStats(int n, int trials)    // perform trials independent experiments on an n-by-n grid
-   public double mean()                          // sample mean of percolation threshold
-   public double stddev()                        // sample standard deviation of percolation threshold
-   public double confidenceLo()                  // low  endpoint of 95% confidence interval
-   public double confidenceHi()                  // high endpoint of 95% confidence interval
-
-   public static void main(String[] args)        // test client (described below)
+    
+    // holds each experiment's percolation threshold result
+   private double[] thresholdResults;
+   
+   private int T;
+   // perform T independent computational experiments on an n-by-n grid
```

Comment:
This is not a well formatted javadoc comment and please could you place it closer to PercolationStats